Sources/PayPalUI/PayPalPayLaterButton.swift

```diff
@@ -5,7 +5,7 @@ import SwiftUI
 public final class PayPalPayLaterButton: PaymentButton, UIViewRepresentable {
 
     /// Available colors for PayPalPayLaterButton.
-    public enum Color: String {
+    public enum Color: String, CaseIterable {
```

Comment:
Does `CaseIterable` have impact to merchant integrations? I'm not sure we want to change the public facing API in order to have our demo app be cleaner

Reply:
This is also on NXO, I took it away at first because i didnt know what it was. Then doing the demo i realized, and IMO it it does create a better feature, by allowing them to iterate over the values, and keeps us better aligned with NXO.

Reply:
+1. I don't see a use case where iterating over color enums would be necessary for merchants. We should take this out to minimize the surface of the SDK API.

Reply:
We should follow the [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it) principle here. We've had experience in the past where the Demo was too tightly coupled to the BT SDK. It's a good rule of thumb to avoid tight coupling like this because the merchant wins in the end.